plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/aps/system/services/content/model/Content.java

```diff
@@ -168,7 +169,7 @@ public void setDefaultModel(String defaultModel) {
     @Override
     public IApsEntity getEntityPrototype() {
         Content content = (Content) super.getEntityPrototype();
-        content.setStatus(STATUS_NEW);
+        content.setStatus(STATUS_DRAFT);
```

Comment:
The status of the prototype has to be "new", draft is the first status that the editor can apply to a content